tpgtools/file_utility.go

```diff
@@ -0,0 +1,24 @@
+package main
```

Comment:
What makes these `main` instead of `utils`?

Reply:
utils is meant for shared functionality across packages. File access isn't something shared so it's silo'd to the main package.